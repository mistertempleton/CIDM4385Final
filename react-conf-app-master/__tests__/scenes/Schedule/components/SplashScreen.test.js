// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import SplashScreen
  from '../../../../app/scenes/Schedule/components/SplashScreen';

describe('Schedule - Splash Screen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SplashScreen />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders correctly in animated state', () => {
    const tree = renderer.create(<SplashScreen animated />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
