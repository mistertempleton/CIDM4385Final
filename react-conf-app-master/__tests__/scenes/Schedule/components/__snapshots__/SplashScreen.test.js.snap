exports[`Schedule - Splash Screen renders correctly 1`] = `
<View
  style={
    Object {
      "height": 1534,
      "position": "absolute",
      "width": 750,
      "zIndex": 2,
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 1734,
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": -200,
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "transform": Array [
              Object {
                "translateY": 0,
              },
              Object {
                "scale": 1,
              },
            ],
            "zIndex": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Image
        source={1} />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(36, 31, 32, 0.8)",
          "bottom": 40,
          "height": 1200,
          "left": -100,
          "position": "absolute",
          "right": -100,
          "transform": Array [
            Object {
              "skewY": "5deg",
            },
          ],
        }
      } />
    <View
      style={
        Object {
          "backgroundColor": "rgba(36, 31, 32, 0.8)",
          "bottom": 40,
          "height": 1200,
          "left": -100,
          "position": "absolute",
          "right": -100,
          "transform": Array [
            Object {
              "skewY": "-3deg",
            },
            Object {
              "translateY": -5,
            },
          ],
        }
      } />
  </View>
</View>
`;

exports[`Schedule - Splash Screen renders correctly in animated state 1`] = `
<View
  style={
    Object {
      "height": 1534,
      "position": "absolute",
      "width": 750,
      "zIndex": 2,
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 1734,
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": -200,
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "transform": Array [
              Object {
                "translateY": 0,
              },
              Object {
                "scale": 1,
              },
            ],
            "zIndex": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Image
        source={1} />
    </View>
    <View
      style={
        Object {
          "backgroundColor": "rgba(36, 31, 32, 0.8)",
          "bottom": 40,
          "height": 1200,
          "left": -100,
          "position": "absolute",
          "right": -100,
          "transform": Array [
            Object {
              "skewY": "5deg",
            },
          ],
        }
      } />
    <View
      style={
        Object {
          "backgroundColor": "rgba(36, 31, 32, 0.8)",
          "bottom": 40,
          "height": 1200,
          "left": -100,
          "position": "absolute",
          "right": -100,
          "transform": Array [
            Object {
              "skewY": "-3deg",
            },
            Object {
              "translateY": -5,
            },
          ],
        }
      } />
  </View>
</View>
`;
